service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationProcessors.java

```diff
@@ -228,8 +234,14 @@ public static Processor slackEventPropertyProcessor(
 
   public static String getSlackMessage(MasProcessingObject mpo, String originalMessage) {
     String msg = originalMessage;
+    EventReason reason = EventReason.getEventReason(originalMessage.trim());
```

Comment:
This seems hacky, i.e., using an _arbitrary_ message string to look up an enum. Shouldn't the reason (`sufficientForFastTracking`, `claimPayload.offRampReason`) be available as part of the `mpo` object, so you can look up the enum based on the contents of the `mpo` object?

Reply:
Yes. It is not architected to have a consistent way of handling events and messages. At this stage, this is a solution. The MAX_CODE_LENGTH = 50 is for this to avoid long string comparison.